GliaWidgets/Localization.swift

```diff
@@ -315,6 +315,32 @@ internal enum Localization {
       /// Chat
       internal static var title: String { Localization.tr("Localizable", "engagement.chat.title", fallback: "Chat") }
     }
+    internal enum Confirm {
+      /// Please allow the {companyName} representative to view this application screen for improved support.
+      internal static var message: String { Localization.tr("Localizable", "engagement.confirm.message", fallback: "Please allow the {companyName} representative to view this application screen for improved support.") }
+      /// Before We Continue
+      internal static var title: String { Localization.tr("Localizable", "engagement.confirm.title", fallback: "Before We Continue") }
+      internal enum Link1 {
+        /// Terms and Conditions
+        internal static var text: String { Localization.tr("Localizable", "engagement.confirm.link1.text", fallback: "Terms and Conditions") }
+        /// http://some_company.com/terms_and_conditions
+        internal static var url: String { Localization.tr("Localizable", "engagement.confirm.link1.url", fallback: "http://some_company.com/terms_and_conditions") }
```

Comment:
Should the fallback have a link like this? Perhaps the fallback can be empty

Reply:
You are right. Fixed 74a7b56a3323de5dea317d5a30da5de8e067303f